pkg/peer/config/service.go

```diff
@@ -24,6 +24,10 @@ import (
 
 var logger = flogging.MustGetLogger("sidetree_peer")
 
+const (
+	defaultMaxBlockchainTransactionsInResponse = 50
```

Comment:
we should also consider a maximum the server will allow to be requested.

i was also wondering how the caller will handle only getting a subset of transactions from a particular time. do we need to more advanced paging or “round” the maxTransactions to complete the time?

Reply:
e.g., it seems clients need the ability to request starting at a certain transaction, in addition to time.

Reply:
This is implemented in the next PR: #217